src/south/Modbus/Modbus.class.js

```diff
@@ -4,6 +4,21 @@ const net = require('net')
 const { getOptimizedScanModes } = require('./config/getOptimizedConfig')
 const ProtocolHandler = require('../ProtocolHandler.class')
 
+/**
+ * Read point value according to it's data type (UInt16, UInt32, etc)
+ * @param {Object} response Response of the modbus request
+ * @param {Object} point The point to read
+ * @param {Number} position Position of the point in the response
+ * @return {Number} Value stored at the specified address
+ */
+const readRegisterValue = (response, point, position) => {
+  if (response.body.constructor.name === 'ReadCoilsResponseBody' || response.body.constructor.name === 'ReadDiscreteInputsResponseBody') {
+    return response.body.valuesAsArray[position]
+  }
+  const funcName = `read${point.dataType}BE`
```

Comment:
If I follow our discussions, the 'BE' part will be another parameters to specify Big or Little endian ?

Reply:
> 
> 
> I think the UI can be improved :
> ![image](https://user-images.githubusercontent.com/28055217/110110559-4d073980-7daf-11eb-8011-8915e4cef817.png)
> 
> I would put the fields in the following order : pointId, address, ModbusType, dataType and scanMode, with more space for pointId, address and scanMode so we can see right away the content of these fields. Can you adapt it please ?

i change the order of the fields, but when I add the "md" parameters on the differents fields of the points schema, the fields ignore this value and stay at the same size. Do you have an idea why ?

Reply:
Yes, I think I'll add an other OIBSelect field to choose between BE and LE